scripts/policy/frameworks/cluster/backend/zeromq/main.zeek

```diff
@@ -139,6 +142,17 @@ export {
 	## for more details.
 	const log_rcvbuf: int = -1 &redef;
 
+	## Set ZMQ_IPV6 option.
+	##
+	## The ZeroMQ library has IPv6 support in ZeroMQ. For Zeek it
+	## we enable it unconditionally such that listening or connecting
+	## with IPv6 just works.
```

Comment:
```suggestion
	## The ZeroMQ library has IPv6 support in ZeroMQ. For Zeek
	## we enable it unconditionally so that listening or connecting
	## with IPv6 just works.
```

And remove the extra blank `##` line below.